src/components/pages/text-curation/edit/table/form/map-locations.tsx

```diff
@@ -0,0 +1,108 @@
+import "leaflet/dist/leaflet.css";
+
+import { Box, Collapse } from "@chakra-ui/react";
+import { LEAFLET_MARKER_ICON } from "@static/constants";
+import L from "leaflet";
+import { LatLngBoundsExpression } from "leaflet";
+import React from "react";
+import { Marker, Tooltip } from "react-leaflet";
+import { MapContainer } from "react-leaflet";
+import { TileLayer } from "react-leaflet";
+
+const onMarkerClickHandler = (
+  row,
+  suggestion,
+  setLatitude,
+  setLongitude,
+  setLocationAccuracy,
+  hForm,
+  locationRef
+) => {
+  setLatitude(suggestion.coordinates[1]);
+  setLongitude(suggestion.coordinates[0]);
+  setLocationAccuracy(suggestion.locationAccuracy);
+  hForm.setValue("longitude", suggestion.coordinates ? suggestion.coordinates[0] : row.longitude);
+  hForm.setValue("latitude", suggestion.coordinates ? suggestion.coordinates[1] : row.latitude);
+  hForm.setValue(
+    "locationAccuracy",
+    suggestion.locationAccuracy ? suggestion.locationAccuracy : row.locationAccuracy
+  );
+  locationRef.current.onChange(
+    {
+      value: suggestion.label,
+      label: suggestion.label,
+      coordinates: suggestion.coordinates,
+      locationAccuracy: suggestion.locationAccuracy
+    },
+    { name: locationRef.current.props.inputId }
+  );
+};
```

Comment:
please add few spaces and if possible small description as well

Reply:
done